featuretools/tests/entityset_tests/test_entity.py

```diff
@@ -22,6 +22,11 @@ def test_is_index_column(es):
     assert es['cohorts'].index == 'cohort'
 
 
+def test_index_at_beginning(es):
```

Comment:
can we add another test? 

I'm thinking create an entity from a DF where the index isn't the first column and make sure that it gets moved to be be the first in the variables list.

Reply:
Yeah makes sense, I'll add that too